deepfence_worker/cronjobs/neo4j.go

```diff
@@ -109,6 +110,18 @@ func CleanUpDB(msg *message.Message) error {
 		return err
 	}
 
+	if _, err = session.Run(`
+		MATCH (n:Node)
+		WHERE n.updated_at < TIMESTAMP()-$time_ms
+		AND n.agent_running=false
```

Comment:
This should be `agent_running=true` here
`agent_running` now means that an agent is deployed, `active` should be used to determine is an agent is active/running.
Maybe we should rename `agent_running` to `agent_present` to avoid such mistakes?